### PR TITLE
605/Fix quick search box

### LINKED ISSUE #605: Quick Search does not work
Using a quick search box (like https://play.google.com/store/apps/details?id=com.startapp.quicksearchbox) no results are shown.

See also #600 for discussion

### PR DESCRIPTION
**Pull request by friedger**
This PR fixes the quick search box behaviour of the app.

With a quick search box like https://play.google.com/store/apps/details?id=com.startapp.quicksearchbox you can jump directly to the desired chapter.

devices

Fixes #605

### CODE REVIEW

**Comment by friedger** [comment]
The country description in the video has been fixed by https://github.com/gdg-x/frisbee/pull/606/commits/54ba81abac413bcdba79a2409b8012659217b6b2

File: app/build.gradle [hunk 1]
```diff
@@ -71,6 +71,7 @@ android {
             buildConfigField "String", "IP_SIMPLE_API_ACCESS_KEY", "\"$local_properties.ip_simple_api_access_key\""
             buildConfigField "String", "ANDROID_SIMPLE_API_ACCESS_KEY", "\"$local_properties.android_simple_api_access_key\""
             resValue "string", "android_simple_api_access_key", local_properties.android_simple_api_access_key ?: ""
+            resValue "string", "authority", "org.gdg.frisbee.android.provider"
```
Comment (tasomaniac) [inline app/build.gradle]: You can also use applicationId in manifest.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -195,7 +195,7 @@
 
     <provider
       android:name=".provider.GdgProvider"
-      android:authorities="${applicationId}.provider.GdgProvider"
+      android:authorities="${applicationId}.provider"
```
Comment (tasomaniac) [inline app/src/main/AndroidManifest.xml]: Oh, you are already using one.

**Comment by tasomaniac** [comment]
:+1:

**Pull request merged**